week6.1-ex1.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Set, Iterable
+from time import time
+
+WORDS_FILE_PATH = r"C:\Users\97252\python\Notebooks\week06\resources\words.txt"
+word_to_search = "zwitterion"
+times_to_search = 1000
+
+
+def create_list_from_file() -> List:
+    """
+    The function open the file and create list of the words that it contains.
+    :return: The created list.
+    """
+    with open(WORDS_FILE_PATH, 'r') as words:
+        words_list = [line for line in words]
+    return words_list
+
+
+def create_set_from_file() -> Set:
+    """
+    The function open the file and create set of the words that it contains.
+    :return: The created set.
+    """
+    with open(WORDS_FILE_PATH, 'r') as words:
+        words_set = [line for line in words]
+    return set(words_set)
+
+
+def average_runtime(words: Iterable) -> float:
+    """
+    The function calculates the average time to search a word in the list/set received multiple times.
```

Comment:
The function calculates the average time to search a word in the recieved list/set.
(multiple times - its a part of the average calculation).